modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-pvcs-to-store-default-os-images_{context}"]
+= Creating ProvisionedVolumeClaims to store default OS images
```

Comment:
**Persistent**VolumeClaims, not **provision**volumeclaims

Reply:
Fixed.